docs/general/universal-profiles.md

```diff
@@ -79,10 +79,25 @@ Removing profiles [is planned](./feature-requests.md) for upcoming releases of t
 
 :::
 
-## Why are received assets not added to my Universal Profile?
-
-If your assets are not getting added to your Universal Profile, it means that the Universal Receiver is not able to edit the storage elements of your Universal Profile. **To solve this issue**, you will need access to your Universal Profile and a second Universal Profile on the same network.
-
-First, please open the original profile within the Universal Profile Browser Extension and [re-apply the default permissions](../extension/controllers.md) to the Universal Receiver's controller.
-
-After the permissions have been renewed, transfer the asset to the second Universal Profile and back to the original profile. The transfer will automatically trigger the Universal Receiver, which will automatically add it to its received assets.
+## My Tokens or NFTs are missing from universalprofile.cloud, how do I add them?
+
+If your assets are not getting added to your Universal Profile, it is likely that the Universal Receiver is not able to edit the storage elements of your Universal Profile.
+
+To solve this issue, [watch this video](https://www.loom.com/share/ff12158265e6482d837d5f9d420b87dd) or follow the steps below:
+
+1. Open the UP! extension, click the "controllers" tab
```

Comment:
I would stick to the official term `Universal Profile Browser Extension`, to have consistency across all pages.

```suggestion
1. Open the Universal Profile Browser Extension and click the "controllers" tab
```